### ISSUE TITLE
Arrow Keys and Mouse Sending Characters Instead of Expected Input

### ISSUE THREAD

**Issue by dammitcoetzee** [tombh/novim-mode#3]
As soon as this plugin is enabled everything gets weird. I can't use a lot of the commands. 
For example Alt-; doesn't do anything. 
Also my arrow keys place "COCOCOBOAODOAODOCOBODOAOCODOCOBODOAOCODOAOCOAOAOAOA" on the screen instead of the expected behavior.

Using vanilla vim on ubuntu 17. 

Fairly excited about this plugin. I would like to help, but I don't know enough about vim to help debug.

**Comment by tombh** [comment]
Thanks a lot for reporting this. I've just been using Neovim in the terminal, so never came across this behaviour. So there are a few things going on here. Firstly there was a bug that was causing the arrow keys to output the `COAB` stuff, that's fixed now and pushed. But in researching this I also discovered the differences of original Vim in the terminal and I forgot, it is has terrible support for key combinations. So now I've added a note in the README about using an extra plugin to get those key combinations back, namely: [vim-fixkey](https://github.com/drmikehenry/vim-fixkey). However, as I also noted, I would actually recommend Neovim over vanilla Vim, if you want a Vim in the terminal, because it has more modern key combination support. In fact Neovim in the terminal is my main dev environment and has been for nearly a year now.

I also added some other recommendations in the README about disabling terminal suspension and getting `SHIFT+ARROWS` working under `tmux`.
